plugins/modules/system/sap_task_list_execute.py

```diff
@@ -0,0 +1,373 @@
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: sap_task_list_execute
+short_description: Perform SAP Task list execution.
+version_added: "3.5.0"
+description:
+  - The C(sap_task_list_execute) module depends on C(pyrfc) Python library (version 2.4.0 and upwards).
+    Depending on distribution you are using, you may need to install additional packages to
+    have these available.
+  - Tasks in the task list which requires manual activities will be confirmed automatically.
+  - This module will use the RFC Package C(STC_TM_API).
+
+options:
+  conn_username:
+    description: The required username for the SAP system.
+    required: true
+    type: str
+  conn_password:
+    description: The required password for the SAP system.
+    required: true
+    type: str
+  host:
+    description: The required host for the SAP system. Can be either an FQDN or IP Address.
+    required: true
+    type: str
+  sysnr:
+    description:
+      - The system number of the SAP system.
+      - This value must be single quoted. Otherwise ansible treated this like a number and replace leading zeros.
+      - Defaults to C('01')
+    default: '00'
+    type: str
+  client:
+    description:
+      - The client number to connect to.
+      - This value must be single quoted. Otherwise ansible treated this like a number and replace leading zeros.
+      - Defaults to C('001')
+    default : '000'
+    type: str
+  task_to_execute:
+    description: The task list which will be executed.
+    required: true
+    type: str
+  task_parameters:
+    description:
+      - The tasks and the parameters for execution.
+      - If the task list do not need any parameters. This could be empty.
+      - The list values must be dictionaries
+        and contain the following keys in uppercase C(TASKNAME), C(FIELDNAME), C(VALUE).
+        See examples.
+      - If only specific tasks from the task list should be executed.
+        The tasks even when no parameter is needed must be provided.
+        Alongside with the module parameter C(task_skip=true).
+    type: list
+    elements: dict
+    suboptions:
+      TASKNAME:
+        description: The name of the task in the task list.
+        type: str
+        required: true
+      FIELDNAME:
+        description: The name of the field of the task.
+        type: str
+      VALUE:
+        description: The value which have to be set.
+        type: raw
+  task_settings:
+    description:
+      - Setting for the execution of the task list. This can be the following as in TCODE SE80 described.
+          Check Mode C(CHECKRUN),
+          Background Processing Active C(BATCH) (this is the default value),
+          Asynchronous Execution C(ASYNC),
+          Trace Mode C(TRACE),
+          Server Name C(BATCH_TARGET).
+    default: ['BATCH']
+    type: list
+    elements: str
+  task_skip:
+    description:
+      - It is possible for tasks that they don't need parameters and run either.
```

Comment:
Not entirely clear to me what this parameter does. Could you please add an example in the `EXAMPLES` blob below?

Reply:
Hi @russoz  I described it a bit better an example could not explain it very well. The prupose for this paramter is if you want to run only specific or certain tasks from the task list.